addon/components/o-s-s/currency-input.ts

```diff
@@ -18,17 +20,25 @@ export default class OSSCurrencyInput extends Component<OSSCurrencyInputArgs> {
   private _currencies = usedCurrencies;
   @tracked currencySelectorShown: boolean = false;
   @tracked filteredCurrencies: Currency[] = this._currencies;
-  @tracked localValue: number = this.args.value || 0;
+  @tracked localValue: number = this.args.value;
 
   constructor(owner: unknown, args: OSSCurrencyInputArgs) {
     super(owner, args);
 
+    if (!this.args.value && !this.args.placeholder) {
+      this.localValue = 0;
+    }
+
```

Comment:
this is to avoid breaking current usages of the component. this way, we skip the default value only if there is a placeholder provided

Reply:
^^ okay !